.github/workflows/release.yaml

```diff
@@ -0,0 +1,31 @@
+name: release
+
+on:
+  push:
+    tags:
+    - 'release-**'
```

Comment:
Currently we sometimes tag main and then create a release branch. We should think about whether we stop this as part of the release.

Reply:
I switched to using `v*` for this workflow to trigger.

Reply:
I like this solution - we can still do the release branch and build image while the binaries are building after the push.